packages/tabs/src/types.ts

```diff
@@ -6,57 +6,61 @@
  */
 
 import { IUseSelectionProps, IUseSelectionReturnValue } from '@zendeskgarden/container-selection';
-import { HTMLProps, ReactNode, RefObject } from 'react';
+import { HTMLProps, ReactNode } from 'react';
 
-export interface IUseTabsProps<Item> extends Omit<IUseSelectionProps<Item>, 'direction'> {
+export interface ITab<Value> {
+  value: Value;
+  disabled?: boolean;
+}
+
+export interface IUseTabsProps<Value>
+  extends Omit<IUseSelectionProps<Value>, 'direction' | 'values'> {
+  /** Provides an ordered list of unique selection values */
+  tabs: ITab<Value>[];
```

Comment:
Please JSDoc this so that consumers are able to grok the internals. Ref: https://github.com/zendeskgarden/react-containers/blob/3676836c2463bee4163139299f156bffa7d26a39/packages/combobox/src/types.ts#L44-L53